tests/test_models.py

```diff
@@ -1,20 +1,20 @@
-#/usr/bin/env python3
+# /usr/bin/env python3
```

Comment:
we should make this line consistent across all tests -- either remove it or correct it (to `#!/usr/bin/env python3`) (if the script is executable and you just run `./test_models.py`, the `#!` line tells your shell which python to use -- not sure it's necessary)

Reply:
Shouldn't it be `pytest test_models.py` instead of `python test_models.py` anyway? I don't see the point of specifying the executable.

Reply:
I agree, that's correct. I'll remove these.